sdk/servicebus/service-bus/src/serializers/queueResourceSerializer.ts

```diff
@@ -31,6 +31,13 @@ import {
  */
 export function buildQueueOptions(queue: CreateQueueOptions): InternalQueueOptions {
   return {
+    // NOTE: this ordering is extremely important. As an example, misordering of the ForwardTo property
+    // resulted in a customer bug where the Forwarding attributes appeared to be set but the portal was
+    // not picking up on it.
+    //
+    // The authority on this ordering is here:
+    // https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/QueuePropertiesExtensions.cs#L20
```

Comment:
Thanks for adding the link! Should it be a permalink in case their repo structure ever changes/a refactor occurs?
> https://github.com/Azure/azure-sdk-for-net/blob/ab9962ee17ea84f3be1aa7f00a0e1354d7c7affc/sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/QueuePropertiesExtensions.cs#L20

Reply:
Yeah, I can do that. My initial thought was just to make sure that it was treated as a "living document" but I think anyone will be able to figure it out.

Reply:
permalinked.